pkg/cmd/oauth-apiserver/openapiconfig/openapiconfig.go

```diff
@@ -33,37 +30,24 @@ func DefaultOpenAPIConfig() *openapicommon.Config {
 		GetDefinitions:    openapigenerated.GetOpenAPIDefinitions,
 		IgnorePrefixes:    []string{"/swaggerapi", "/healthz", "/controllers", "/metrics", "/version/openshift", "/brokers"},
```

Comment:
not sure if `"/version/openshift", "/brokers"` are still valid

Reply:
and why not /readyz and /livez ?